live-configurator/doc/live-configurator.md

```diff
@@ -1,6 +1,6 @@
 ```js script
 import '~/doc-layout';
-import '../src/fs-live-configurator.js';
+import '../src/fs-live-configurator';
```

Comment:
if you would ask me how to fix the inconsistency in the file, I'd say let's add extensions in all JS files, including the doc-layout... this doesn't seem right to me, it's not a TypeScript context

is it a temp fix or will always be like that in Vite?

Reply:
No it was temporary sorry for this useless PR